nova/virt/vmwareapi/vmops.py

```diff
@@ -1750,9 +1756,18 @@ def poll_rebooting_instances(self, timeout, instances):
 
     def get_info(self, instance):
         """Return data about the VM instance."""
-        vm_ref = vm_util.get_vm_ref(self._session, instance)
 
         lst_properties = ["runtime.powerState"]
+
+        if instance.uuid in vm_util._VM_VALUE_CACHE:
```

Comment:
The `_VM_VALUE_CACH` should be keyed by `vm_ref`.
So, the lookup should happen after (1769)